tests/testthat/helper.R

```diff
@@ -62,3 +62,19 @@ defer <- function(expr, frame = parent.frame(), after = FALSE) {
   thunk <- as.call(list(function() expr))
   do.call(on.exit, list(thunk, TRUE, after), envir = frame)
 }
+
+# always returns a named list
+nlist <- function(...) {
```

Comment:
Might be better to make this more specific? Like:

```R
empty_named_list <- function() setNames(, list())
```

Or since this is just annoying, it might be better to ensure that the underlying code returns an unnamed list when length 0?

Reply:
I modified `constructor_args()` so it returns pairlists. A length-0 `pairlist()` evaluates to `NULL`, so we no longer need a helper to create an empty named list. I removed the helper.